system/Helpers/filesystem_helper.php

```diff
@@ -290,7 +290,7 @@ function get_filenames(string $source_dir, ?bool $include_path = false, bool $hi
 	function get_dir_file_info(string $source_dir, bool $top_level_only = true, bool $recursion = false): array
 	{
 		static $fileData = [];
-		$relative_path   = $source_dir;
+		$relativePath    = $source_dir;
```

Comment:
I wonder if there is a rule that will allow us to rename parameters for functions as well?

Reply:
there is https://github.com/rectorphp/rector/blob/master/rules/coding-style/src/Rector/Variable/UnderscoreToCamelCaseVariableNameRector.php for that, but we will need to update documentation: doc block and user guide manually

Reply:
Ok, great. I think we should add this too.

Updating the user guide probably will take some time but it's not that it will change much for users... hopefully, people can figure out the difference quite easy :)

Reply:
Updating docblock `@param` manually in the code will take time as well, as phpstan will report that. I will check if I can add auto-update `@param` docblock rule when updating parameter value if I have a chance

Reply:
Yes, you're right. I forgot about phpstan.

Reply:
implemented in latest commit.